v6/jbuild

```diff
@@ -32,10 +32,25 @@ let () = Printf.ksprintf Jbuild_plugin.V1.send {|
  ((name xcp_v6)
   (public_name xcp.v6)
   (flags (:standard -w -39 %s))
+  (modules (:standard \ v6_cli ))
   (libraries
    (rpclib
     threads
     xcp))
   (wrapped false)
   %s))
+
+(executable
+ ((name v6_cli)
+  (modules (v6_cli))
+  (libraries
+   (cmdliner
+    rpclib.cmdliner
+    rpclib.markdown
+    xcp.v6))))
+
+(alias
+ ((name runtest)
+  (deps (v6_cli.exe))))
```

Comment:
This doesn't make sense. Is it actually used in the tests?

You can build it already by doing `jbuilder build v6/v6_cli.exe` and run it with `_build/default/v6/v6_cli.exe` (and soon with `jbuilder exec ./v6/v6_cli.exe`)

Reply:
[Can I just say that is possibly the fastest response to a PR I've ever had?]

I just checked, and you are correct! The difference is that including these lines tells `make test` to build `v6_cli.exe`, whereas without, it currently builds the following:
```
lib/channel_helper.exe
lib_test/test.exe
memory/memory_cli.exe
example/example.exe
```
I think the question is: do we want to include the daemon CLIs for `make test`, or have them explicitly built? If the former, we probably shouldn't build the squeezed CLI either, and if the latter, we should perhaps document the CLI a little more explicitly on our side so devs know about it. What do you think?

Reply:
As we discussed, let's remove all of them, and let's run the example in the tests.